src/el/el_client_context.star

```diff
@@ -7,6 +7,7 @@ def new_el_client_context(
 	ws_port_num,
 	engine_rpc_port_num,
 	jwt_secret,
+	metric_url,
```

Comment:
can we make this the last thing we pass and put `""` at the end, this ensures backwards compatibilty with any other callers of this function that haven't started passing metric_url,